### PR TITLE
Feature/ocserv version autoupdate

### PR DESCRIPTION
**Pull request by framebassman**
https://github.com/aminvakil/docker-ocserv/settings/actions
image

### CODE REVIEW

**Comment by framebassman** [comment]
@aminvakil 
I suggest to merge this one. After that I will create the PR in nvchecker (it may take some time because it is not so obvious) and we could revert this one

File: .github/workflows/ocserv-version-update.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      contents: write
+      pull-requests: write
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Update openconnect version
+      id: get_version
+      run: |
+        OC_VERSION=$(curl https://gitlab.com/api/v4/projects/openconnect%2Focserv/repository/tags | jq -r '.[0].name')
```
Comment (aminvakil) [inline .github/workflows/ocserv-version-update.yml:19]: Let's change this to the same url we use in Dockerfile.
```suggestion
        OC_VERSION=$(curl -s ftp://ftp.infradead.org/pub/ocserv/ | awk '{print $9}' | tail -n 1 | sed 's:.*-::;s:.tar.*::')
```

File: .github/workflows/ocserv-version-update.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+name: bump ocserv version
+on:
+  schedule:
+  - cron: "0 0 * * *"
+jobs:
+  ocserv-version-update:
+    runs-on: ubuntu-latest
+    permissions:
```
Comment (aminvakil) [inline .github/workflows/ocserv-version-update.yml:8]: Do we need this here?

File: .github/workflows/ocserv-version-update.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+name: bump ocserv version
+on:
```
Comment (aminvakil) [inline .github/workflows/ocserv-version-update.yml:2]: Please add pull_request here to see this run prior to merging it.

File: .github/workflows/ocserv-version-update.yml (hunk 2 above)
Comment (framebassman) [reply .github/workflows/ocserv-version-update.yml:9]: Yep - this is a result without the permission section - https://github.com/framebassman/docker-ocserv/actions/runs/7519894802/job/20468951183

**Comment by aminvakil** [comment]
Please do not force-push on every commit, it makes reviewing harder, we will squash and fix commits at last.

**Review by aminvakil — APPROVED** [review]

**Comment by aminvakil** [comment]
Thank you for doing this!

I will open a PR whenever I found some time (and if you didn't beat me to it) in https://github.com/aminvakil/nvchecker later on, so this bumping would be centralized and could be used for other stuff as well.

Then this will be reverted.

But until then, this would definitely be an improvement, thanks again!

**Pull request merged**